src/frontend/ts/state/chessStore.ts

```diff
@@ -34,6 +60,15 @@ const useChessStore = create<ChessState>((set, get) => ({
       event: (data: ChessUpdate) => get().receiveUpdate(data),
       quit: () => {}
     })
+
+    const opp: string = data.white === window.ship ? data.black : data.white 
+    if (get().outgoingChallenges.has(opp)) {
+      if (data.event === get().outgoingChallenges.get(opp).event) {
+        // XX: should this take into account our and our opponent's
+        //     sides to narrow it down further?
+        get().removeOutgoingChallenge(opp)
+      }
+    }
```

Comment:
This removes outgoing challenges when they're accepted by our opponent. I don't love it, but the first alternative that comes to mind is giving every challenge its own ID, which seems like overkill.

Reply:
Another option: change default `event` from 'random', 'black', or 'white' to something like '~2022.12.7'. This would avoid collision of default event names, but still feels a bit clumsy.